hivemind_summarizer/activities.py

```
@@ -272,6 +219,131 @@ async def fetch_platform_summaries_by_date_range(
+        logging.info(f"found {len(search_results)} raw data points!")
+
+        # Extract text content from the results
+        texts = []
+        for point in search_results:
+            if "_node_content" in point.payload:
+                content = point.payload["_node_content"]
+                if isinstance(content, str):
+                    try:
+                        content = json.loads(content)
+                    except json.JSONDecodeError:
+                        pass
```

Ruff (0.8.2)

Use `contextlib.suppress(json.JSONDecodeError)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(json.JSONDecodeError)`

(SIM105)